pkgs/applications/networking/sbot/default.nix

```diff
@@ -0,0 +1,738 @@
+{ stdenv, nodePackages }:
+
+# to update sbot, run npm2nix package.json package.nix
+
+let
+	np = nodePackages.override {
+		generated = ./generated.nix;
+	};
+in
+
+np.buildNodePackage rec {
+	version = "9.4.2";
+  name = "sbot-${version}";
```

Comment:
Couple things,

1. you're actually doing `init at 9.4.2` -- can you update the commit message?
2. can you dedent `version` back to the normal level here?